src/MockWebServer.php

```diff
@@ -282,4 +273,29 @@ private function findOpenPort() {
 
 		throw new Exceptions\RuntimeException('Failed to find open port');
 	}
+
+	private function isWindowsPlatform() {
+		return defined('PHP_WINDOWS_VERSION_MAJOR');
+	}
+
+	private function startServer($fullCmd) {
+		if( !$this->isWindowsPlatform() ) {
+			// We need to prefix exec to get the correct process http://php.net/manual/ru/function.proc-get-status.php#93382
+			$fullCmd = 'exec ' . $fullCmd;
+		}
+		$pipes = [];
+		$env = null;
+		$cwd = null;
+		$process = proc_open($fullCmd, [], $pipes, $cwd, $env, [
+			'suppress_errors' => false,
+			'bypass_shell' => true,
+		]);
+
+		sleep(1);
+
+		if( !is_resource($process) ) {
+			return false;
```

Comment:
I'd say rather than returning false here, we move the throw from line 81 into here, such that the return of this method is always guaranteed to be a resource.

Reply:
I agree because  of proc_open returns a process if successful, otherwise false